nimble/host/src/ble_l2cap.c

```diff
@@ -387,6 +387,15 @@ ble_l2cap_rx(struct ble_hs_conn *conn,
             goto err;
         }
 
+        if (l2cap_hdr.len > chan->my_coc_mps) {
```

Comment:
what about ATT ? in such a case chan->my_coc_mps is 0 I believe. Or not?

Reply:
Yes, I think you are right. What's more, I think only dynamic range CIDs use `my_coc_mps` field, and for others MPS=MTU of channel, which varies from channel to channel. I added 3 cases for `BLE_L2CAP_CID_ATT`, `BLE_L2CAP_CID_SIG` and `BLE_L2CAP_CID_SM`.